src/test/scala/org/labrad/TlsTest.scala

```diff
@@ -2,23 +2,23 @@ package org.labrad
 
 import java.io.File
 import org.labrad.data._
-import org.labrad.manager.TlsPolicy
+import org.labrad.manager.{ManagerUtils, TlsPolicy}
 import org.labrad.types._
 import org.scalatest.FunSuite
 import scala.concurrent.Await
 import scala.concurrent.duration._
 
 class TlsTest extends FunSuite {
 
-  import TestUtils._
+  import ManagerUtils._
 
   def connectWithMode(tls: TlsMode, includeCerts: Boolean = true)(implicit m: ManagerInfo): Unit = {
     val certs = if (includeCerts) {
       Map(m.host -> m.tlsHosts.certFiles.map(m.host))
     } else {
       Map.empty[String, File]
     }
-    withClient(m.host, m.port, m.credential, tls = tls, tlsCerts = certs) { c =>
+    withClient(m, tls = tls, tlsCerts = certs) { c =>
```

Comment:
Does it make sense to put the TLS parameters into the ManagerInfo class? Or are there situations where you'd use TLS outside of interacting with the manager?